server/chat.js

```diff
@@ -286,7 +286,17 @@ Chat.statusfilters = [];
  * @param {User} user
  */
 Chat.statusfilter = function (status, user) {
+	// Protocol character
 	status = status.replace(/\|/g, '');
+	if (!Config.disablebasicnamefilter) {
+		// e-mail address
+		if (status.includes('@') && status.includes('.')) return '';
+		// url
+		if (/[a-z0-9]\.(com|net|org|us|uk|co|gg|tk|ml|gq|ga|xxx|download|stream|)\b/.test(status)) status = status.replace(/\./g, '');
```

Comment:
Also why not?

Reply:
ditto, see the guy in #5569

Reply:
I don't really have a problem with people advertising in a status. It's meant for you to talk about yourself.

Reply:
Isn’t it odd that advertising in PMs and rooms will get you locked but not statuses though? It seems like it should be consistent

Reply:
passive vs. active

Reply:
Advertising in your name will get you filtered or FRd, but not statuses?

Reply:
I mean, I think its slightly inconsistent as well, but I was just trying to provide what I believe Zarel's argument would be. However, name/PMs/rooms are all in your face advertisements, you need to go out of your way to see someones status and so its not necessarily the same thing as the others.

`.xxx` domains though... - are ANY of those ones we would want in someones status lol?

Reply:
`.xxx` domains should already be filtered out by the other filters.

Reply:
Advertising in private rooms will not get you locked.

Neither will advertising to your friends in PMs.

I consider status messages roughly on the level of those.

Reply:
In particular, there's a difference between advertising _after_ someone asks, "I want to hear what you have to say", and advertising to someone who did not and has no intention to. The purpose of the status message is advertising. Basically any use of the status message is against chat rules. I would also lock you if you spammed "I'm going afk to eat dinner" in PM to a bunch of people who weren't talking to you, and you would at least get a warning if you said that in most public rooms you weren't actively participating in.

Heck, if I ask someone in a public room "what are you working on?" and they say "I'm programming for this private server asdf.psim.us", I would be an asshole if I locked them. That's not advertising.